modules/claims_api/app/controllers/claims_api/v2/veterans/claims_controller.rb

```diff
@@ -68,6 +70,7 @@ def generate_show_output(bgs_claim:, lighthouse_claim:) # rubocop:disable Metric
           structure.merge!(errors: get_errors(lighthouse_claim))
           structure.merge!(supporting_documents: build_supporting_docs(bgs_claim))
           structure.merge!(tracked_items: map_bgs_tracked_items(bgs_claim))
+          structure.merge!(build_additional_bgs_attributes(bgs_claim))
```

Comment:
This should really be named something more specific like, `build_claim_phase_attributes()`.